Lib/test/test_json/test_tool.py

```diff
@@ -68,22 +68,22 @@ def test_stdin_stdout(self):
         self.assertEqual(out.splitlines(), self.expect.encode().splitlines())
         self.assertEqual(err, None)
 
-    def _create_infile(self):
+    def _create_infile(self, text):
         infile = support.TESTFN
         with open(infile, "w") as fp:
             self.addCleanup(os.remove, infile)
-            fp.write(self.data)
+            fp.write(text)
```

Comment:
Done in https://github.com/python/cpython/pull/201/commits/a3e2b23b1115e66444469685b57e534f0cef69ff.